DataFormats/Math/test/ExtVec_t.cpp

```diff
@@ -1,25 +1,21 @@
-#include "FWCore/Utilities/interface/GCC11Compatibility.h"
-#if GCC_PREREQUISITE(4,8,0)
-
 #include "DataFormats/Math/interface/ExtVec.h"
 
 #include<cmath>
 #include<vector>
 
 #include<iostream>
 
+#ifdef __AVX__
+#define CMS_USE_AVX
+#endif /* __AVX__ */
 
-// #include <x86intrin.h>
+// clang complains that it is not "standard"
+#define constexpr
```

Comment:
I'd rather the few problematic "constexpr" lines are edited instead of changing the rules of the language because it's apparently easier to do.